packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -165,12 +169,6 @@ function DreifussEditor(props: DreifussWysiwygEditorOptions) {
       : {fontFamily: 'Helvetica'}
   }
 
-  const charCount = getCharacterCount(id)
```

Comment:
this was a reason of different bugs. maybe because we were using  plate custom hook in the Parent component where `<Plate />` is being used instead of a child component